x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.ts

```diff
@@ -304,18 +304,42 @@ export const PARAMETERS_DEFINITION = {
   },
   scaling_factor: {
     fieldConfig: {
-      defaultValue: 1.0,
+      defaultValue: '',
       type: FIELD_TYPES.NUMBER,
       formatters: [toInt],
+      label: i18n.translate('xpack.idxMgmt.mappingsEditor.parameters.scalingFactorLabel', {
+        defaultMessage: 'Scaling factor',
+      }),
       validations: [
+        {
+          validator: emptyField(
+            i18n.translate(
+              'xpack.idxMgmt.mappingsEditor.parameters.validations.scalingFactorIsRequiredErrorMessage',
+              {
+                defaultMessage: 'A scaling factor is required.',
+              }
+            )
+          ),
+        },
         {
           validator: ({ value }: ValidationFuncArg<any, number>) => {
-            if (value < 0) {
-              return { message: commonErrorMessages.smallerThanZero };
+            if (value <= 0) {
+              return {
+                message: i18n.translate(
+                  'xpack.idxMgmt.mappingsEditor.parameters.validations.greaterThanZeroErrorMessage',
+                  {
+                    defaultMessage: 'The scaling factor must be greater than 0.',
+                  }
+                ),
+              };
             }
           },
         },
       ],
+      helpText: i18n.translate('xpack.idxMgmt.mappingsEditor.parameters.scalingFactorHelpText', {
+        defaultMessage:
+          'Values will be multiplied by this factor at index time and rounded to the closest long value.',
```

Comment:
I think we should remove this help text and move it into the description instead. It seems like the help text is better used for describing validation rules. If you agree I'll update https://github.com/elastic/kibana/pull/52363 accordingly since I did something similar there.

Here's what I think would make sense for description text:

> Values will be multiplied by this factor at index time and rounded to the closest long value. High factor values improve accuracy, but also increase space requirements.

Reply:
>  It seems like the help text is better used for describing validation rules. 

This makes sense. I think I recall design suggesting the same.